src/main/java/de/eldoria/pickmeup/commands/Reload.java

```diff
@@ -27,7 +27,7 @@ public Reload(Plugin plugin) {
 
     @Override
     public void onCommand(@NotNull CommandSender sender, @NotNull String alias, @NotNull Arguments args) throws CommandException {
-        getPlugin().onEnable();
+        PickMeUp.instance().onReload();
```

Comment:
And thats once again a reason why I dont like static xD But yeah thats on me.

Replace with:
`((PickMeUp)plugin()).onReload();`

and remove the static import on top from the towny plugin.